### DISCUSSION TITLE
Why does `https://adminlte.io/docs/` redirect to the 2.4 docs instead of 3.0?

### DISCUSSION THREAD

**Discussion by mgrdcm** [ColorlibHQ/AdminLTE#3218] [Q&A] [answered]
Greetings!

I'm new to AdminLTE, but it appears 3 has been the current stable release for a while but the "Docs" link in the header of AdminLTE.io redirects to the previous 2.4 version and 3.0 isn't in the dropdown there.  Is there a reason?

**Comment by REJack** [comment, marked as answer]
Yeah there is a reason, the adminlte.io page is a custom built page and I have no access to change the layout or menu itself. But there will come a new page soon.
